### PR TITLE
[DE] localize Cluster

### PR DESCRIPTION
**Pull request by iamNoah1**
### Describe your changes
A new approach to localize the term Cluster 

### Related issue number or link (ex: `resolves #issue-number`)
#1407

deprecates #1967

Thanks @bjoernschotte for the contribution, this is a new approach. I opened a new issue in order to update the English term accordingly. #2210

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by BjoernSchotte** [comment]
Thanks @iamNoah1 for the excellent cooperation. Really appreciated that.

I assume that English language is the leading branch, so advancements in localized branches have to be pulled back to the English branch in order to achieve better consistency.

Not sure if you evaluated toolings for this (I saw some slack message about this), maybe there are some toolings for l10ns which could help improving consistency.

Further thinking: If there are n:1 PRs re 1 english term (i.e. multiple suggestions from different branches) there will be some coordination effort, too.

**Comment by iamNoah1** [comment]
> Thanks @iamNoah1 for the excellent cooperation. Really appreciated that.
> 
> I assume that English language is the leading branch, so advancements in localized branches have to be pulled back to the English branch in order to achieve better consistency.
> 
> Not sure if you evaluated toolings for this (I saw some slack message about this), maybe there are some toolings for l10ns which could help improving consistency.
> 
> Further thinking: If there are n:1 PRs re 1 english term (i.e. multiple suggestions from different branches) there will be some coordination effort, too.

see #2210

File: content/de/cluster.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Ein weiteres Szenario könnte sein, dass dieser Computer oder die Anwendung darin zwar funktionsfähig sind, 
+aber über das Netzwerk nicht erreichbar sind. 
+
+
+## Wie es das Problem löst
+
+Ein Cluster löst dieses Problem indem mehr als nur ein Computer verwendet wird. 
+Man spricht hier auch von verteilten Systemen. 
+Wenn eines dieser Computer oder die Anwendung darin abstürzt oder aus anderen Gründen ausfällt, können die anderen Computer übernehmen. 
+Hierfür wird üblicherweise eine Komponente verwendet die als Load Balancer bekannt ist und anfragen an ein Cluster lediglich an erreichbare Computer weiterleitet.
```
Comment (DaveVentura) [inline content/de/cluster.md:29]:
```suggestion
Hierfür wird üblicherweise eine Komponente verwendet, die als Load Balancer bekannt ist und Anfragen an ein Cluster lediglich an erreichbare Computer weiterleitet.
```

File: content/de/cluster.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Business-kritische Systeme können offline gehen, wenn dieser Computer oder die Anwendung darin abstürzt. 
+Ein weiteres Szenario könnte sein, dass dieser Computer oder die Anwendung darin zwar funktionsfähig sind, 
+aber über das Netzwerk nicht erreichbar sind. 
+
+
+## Wie es das Problem löst
+
+Ein Cluster löst dieses Problem indem mehr als nur ein Computer verwendet wird. 
+Man spricht hier auch von verteilten Systemen. 
+Wenn eines dieser Computer oder die Anwendung darin abstürzt oder aus anderen Gründen ausfällt, können die anderen Computer übernehmen.
```
Comment (DaveVentura) [inline content/de/cluster.md:28]:
```suggestion
Wenn einer dieser Computer oder die Anwendung darin abstürzt oder aus anderen Gründen ausfällt, können die anderen Computer übernehmen. 
```

**Review by DaveVentura — CHANGES_REQUESTED** [review]

**Comment by iamNoah1** [comment]
@DaveVentura hab die suggestions eingebaut. Können wir noch einen review von @bcubk bekommen bitte :)

**Review by bcubk — APPROVED** [review]
Finde ich sehr gut übersetzt. Das Orginal in Englisch ist schon sehr aufgebläht. Diese Version finde ich sehr schlank und sachlich. LGTM 👍

File: content/de/cluster.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+status: Feedback erwünscht
+category: Konzept
+tags: ["infrastruktur", "grundlegendes", ""]
+---
+
+## Was es ist
+
+Ein Cluster ist eine Gruppe von Computern, die ein gemeinsames Ziel haben und als eine Einheit auftreten.
+Im Kontext von [Cloud Native Technologien](/cloud-native-tech/) wird der Begriff häufig mit [Kubernetes](/kubernetes/) gleichgesetzt.
+Jedenfalls kommunizieren einzelnen Computer über ein geeignetes Netzwerk miteinander.
```
[the remark this answers is not in the document]
Comment (iamNoah1) [reply content/de/cluster.md:12]:
```suggestion
Jedenfalls kommunizieren einzelnen Computer über ein Netzwerk miteinander. 
```

**Pull request merged**